### PR TITLE
added down logic for brutish behavior

### PR DESCRIPTION
**Pull request by maikelmclauflin**

### CODE REVIEW

File: eyeshade/migrations/0000_initial/down.sql [hunk 1]
```diff
@@ -1 +1,5 @@
+select execute($$
+
 drop table migrations;
+
+$$) where exists (select * from migrations where id = '0001');
```
Comment (evq) [inline eyeshade/migrations/0000_initial/down.sql]: should this be 0000?

File: eyeshade/migrations/0003_ads/down.sql [hunk 2]
```diff
@@ -90,3 +92,5 @@ create materialized view account_balances(
 create unique index on account_balances(account_type, account_id);
 
 refresh materialized view account_balances;
+
+$$) where exists (select * from migrations where id = '0004');
```
Comment (evq) [inline eyeshade/migrations/0003_ads/down.sql]: should this be 0003?

File: eyeshade/migrations/0000_initial/down.sql (hunk 1 above)
Comment (maikelmclauflin) [reply eyeshade/migrations/0000_initial/down.sql]: yis

File: eyeshade/migrations/0003_ads/down.sql (hunk 2 above)
Comment (maikelmclauflin) [reply eyeshade/migrations/0003_ads/down.sql]: yis

**Pull request merged**